skycoin/skywire
# Fix/ui rebuild

Did you run `make format && make check`?
Yes

 Changes:	
- rebuilds the frontend with latest changes

How to test this PR:
- setup hypervisor and visor
- check for changes to the frontend (transport status is displayed, history of skysocks client is available)
